### ISSUE TITLE
Translate stock status

### ISSUE THREAD

**Issue by MichaelaKubasova** [CrocoBlock/suggestions#5498]
Ahoj, vytvoril som dvojjazyčný webshop, napísal som s podporou. Ale keď mám slovenskú verziu stránky a preložím stav zásob vo woo builderi, potrebujem upraviť anglickú verziu stránky. Ale s WPML sa nedá preložiť. Malo by to fungovať. Pretože musím vytvoriť webovú stránku s piatimi jazykmi a chcem mať jednu šablónu a upravovať tieto slová vo wpml.

**Comment by Floydtm** [comment]
Please, note, that the feature requests can be accepted and processed only if they’re written in English.

**Comment by MichaelaKubasova** [comment]
Sorry, I try google translate and copy slovak version. 

Hi, I created a bilingual webshop, I wrote with support. But when I have the Slovak version of the site and I translate the inventory in the woo builder, I need to edit the English version of the site. But it can't be translated with WPML. It should work. Because I have to create a website with five languages and I want to have one template and edit these words in wpml.
